addons/2d-color-picker/paint-editor.js

```diff
@@ -38,14 +38,12 @@ export default async ({ addon, console, msg }) => {
     const onEyeDropperOpened = ({ detail }) => {
       if (detail.action.type !== "scratch-paint/eye-dropper/ACTIVATE_COLOR_PICKER") return;
       addon.tab.redux.removeEventListener("statechanged", onEyeDropperOpened);
-      setTimeout(() => {
```

Comment:
Why was the `setTimeout` needed before?

Reply:
apple502j says:
> redux event handler shouldn't dispatch another change
So that's why setTimeout is there in the first place, as far as we know